### DISCUSSION TITLE
About language settings

### DISCUSSION THREAD

**Discussion by bewareilive** [mockoon/mockoon#658] [Help] [answered]
I would like to translate the desktop application UI, then recompile it and recommend it to others.
I don't know if this is possible.
Or is it possible to have a configuration file available for users to do the translation themselves.
hope to get help.

**Comment by 255kb** [comment, marked as answer]
You can surely fork the project, translate the texts and rebuild the app. However, it wouldn't be an official supported build and the binaries wouldn't be signed.
Mockoon is an Electron/Angular app so it wouldn't be complicated to add a translation library. Then, it's a bit of work to extract all the texts from the components and regroup them in a translation file. But I guess doing this would benefit the project greatly, and would open the door to more translations.
